.github/workflows/build_documentation.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       package_name: audio-course
       path_to_docs: audio-transformers-course/chapters/
       additional_args: --not_python_module
-      languages: en
+      languages: en, ben
```

Comment:
as [here](https://github.com/huggingface/course/blob/main/.github/workflows/build_pr_documentation.yml#L19), we just list them without comma 😊